quadlets/filebrowser/filebrowser.container

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Filebrowser Quadlet
+
+[Service]
+Restart=always
+TimeoutStartSec=900
+
+[Install]
+WantedBy=default.target
+
+[Container]
+Image=docker.io/filebrowser/filebrowser:latest
+ContainerName=filebrowser
+HostName=filebrowser
+
+# You will have to manually create the config and db file on first run as it mounts files not dirs
+Volume=/path/to/what/you/want/to/share:/srv:z
+Volume=%h/.local/share/containers/storage/volumes/filebrowser-db/_data/database.db:/database.db:Z
+Volume=%h/.local/share/containers/storage/volumes/filebrowser-config/_data/.filebrowser.json:/.filebrowser.json:Z
+Volume=filebrowser-branding:/branding
+
+PublishPort=8008:8008
+
+Environment=PUID=1000
```

Comment:
These are holdovers from an outmoded method of handling docker user/group stuff inside containers. `podman` should not require this anymore, and `1000` cannot be hardcoded. Even though upstreams and lsio containers use it as an example all the time, we have subuids and subgids, and `1000` is only correct if you are the only user on the system.